### ISSUE TITLE
Incompatible with Meteor 1.6.1-rc.1 due to babel-compiler upgrade

### ISSUE THREAD

**Issue by choudrio** [Urigo/angular-meteor#1855]
Hi @Urigo and @ardatan 
recently Meteor 1.6.1 was upgraded to use babel-compiler 7.0.0 which breaks angular-meteor compatibility.

An example is to upgrade the examples/MeteorCLI/all-in-one and run it with an updated Meteor.

Running AOT=1 meteor using Meteor 1.6.0.1 works.

Error with 1.6.1:
examples/MeteorCLI/all-in-one

meteor update --release 1.6.1-rc.1
=> Errors while initializing project:         
                                              
While selecting package versions:
error: Conflict: Constraint babel-compiler@6.24.7 is not satisfied by babel-compiler 7.0.0-rc161.1.
Constraints on package "babel-compiler":
* babel-compiler@~7.0.0-rc161.1 <- top level
* babel-compiler@7.0.0-rc161.1 <- ecmascript 0.10.0-rc161.1
* babel-compiler@7.0.0-rc161.1 <- standard-minifier-js 2.3.1-rc161.1
* babel-compiler@7.0.0-rc161.1 <- minifier-js 2.3.1-rc161.1 <- standard-minifier-js 2.3.1-rc161.1
* babel-compiler@6.24.7 <- angular-typescript-compiler 0.2.9 <- angular-compilers 0.2.8_1

Thanks for your help!

**Comment by greatGregLiu** [comment]
I met same problem.

**Comment by Slavrix** [comment]
Meteor 1.6.1 is now released, hoping there will be an update soon.

**Comment by filosof86** [comment]
The same issue on 1.6.1

**Comment by HectorRPL** [comment]
The same issue on 1.6.1.
I hope a a solution soon ... because I need for some libraries and frameworks that works with meteor 1.6.1 =(

**Comment by pbureau** [comment]
Hi we have the same issue trying to upgrade to 1.6.1 from version 1.6.0.1

I there any known workaround for this problem?

**Comment by atao60** [comment]
Hi  @choudrio, @pbureau,

When I try to update examples/MeteorCLI/all-in-one from 1.6 to 1.6.0.1, I get:

```
$ svn export https://github.com/Urigo/angular-meteor/trunk/examples/MeteorCLI/all-in-one
$ cd all-in-one
$ meteor npm install
$ meteor update --patch
=> Errors while initializing project:         
                                              
While selecting package versions:
error: Conflict: Constraint babel-compiler@6.24.7 is not satisfied by babel-compiler 7.0.4.
Constraints on package "babel-compiler":
* babel-compiler@~7.0.0 <- top level
* babel-compiler@7.0.0 <- ecmascript 0.10.0
* babel-compiler@7.0.0 <- standard-minifier-js 2.3.1
* babel-compiler@7.0.0 <- minifier-js 2.3.1 <- standard-minifier-js 2.3.1
* babel-compiler@6.24.7 <- angular-typescript-compiler 0.2.9 <- angular-compilers 0.2.8_1

```
As it seems that you managed to update your project, can you share your configuration (at least package.json, .meteor/packages, tsconfig.json)

**Comment by pbureau** [comment]
Hi,
      
      I did not manage to make it work, the only configurations that
      works for me is using Meteor 1.6.0.1 and the old angular2-compiler
      
      Pierre
      
      On 17/02/2018 21:45, atao60 wrote:
    
    
      Hi @choudrio,
        @pbureau,
      When I try to update examples/MeteorCLI/all-in-one from 1.6 to
        1.6.0.1, I get:
      $ svn export https://github.com/Urigo/angular-meteor/trunk/examples/MeteorCLI/all-in-one
$ cd all-in-one
$ meteor npm install
$ meteor update --patch
=> Errors while initializing project:         
                                              
While selecting package versions:
error: Conflict: Constraint babel-compiler@6.24.7 is not satisfied by babel-compiler 7.0.4.
Constraints on package "babel-compiler":
* babel-compiler@~7.0.0 <- top level
* babel-compiler@7.0.0 <- ecmascript 0.10.0
* babel-compiler@7.0.0 <- standard-minifier-js 2.3.1
* babel-compiler@7.0.0 <- minifier-js 2.3.1 <- standard-minifier-js 2.3.1
* babel-compiler@6.24.7 <- angular-typescript-compiler 0.2.9 <- angular-compilers 0.2.8_1


      As it seems that you managed to update your project, can you
        share your configuration (at least package.json,
        .meteor/packages, tsconfig.json)
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Urigo/angular-meteor","title":"Urigo/angular-meteor","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Urigo/angular-meteor"}},"updates":{"snippets":[{"icon":"PERSON","message":"@atao60 in #1855: Hi  @choudrio, @pbureau,\r\n\r\nWhen I try to update examples/MeteorCLI/all-in-one from 1.6 to 1.6.0.1, I get:\r\n\r\n```\r\n$ svn export https://github.com/Urigo/angular-meteor/trunk/examples/MeteorCLI/all-in-one\r\n$ cd all-in-one\r\n$ meteor npm install\r\n$ meteor update --patch\r\n=\u003e Errors while initializing project:
  
        \r\n                                              \r\nWhile selecting package versions:\r\nerror: Conflict: Constraint babel-compiler@6.24.7 is not satisfied by babel-compiler 7.0.4.\r\nConstraints on package \"babel-compiler\":\r\n* babel-compiler@~7.0.0 \u003c- top level\r\n* babel-compiler@7.0.0 \u003c- ecmascript 0.10.0\r\n* babel-compiler@7.0.0 \u003c- standard-minifier-js 2.3.1\r\n* babel-compiler@7.0.0 \u003c- minifier-js 2.3.1 \u003c- standard-minifier-js 2.3.1\r\n* babel-compiler@6.24.7 \u003c- angular-typescript-compiler 0.2.9 \u003c- angular-compilers 0.2.8_1\r\n\r\n```\r\nAs it seems that you managed to update your project, can you share your configuration (at least package.json, .meteor/packages, tsconfig.json)\r\n"}],"action":{"name":"View Issue","url":"https://github.com/Urigo/angular-meteor/issues/1855#issuecomment-366470156"}}}
    
    
    
    -- 
http://www.mykodibox.fr
Mobile: +33 6 74 79 18 59

**Comment by pbureau** [comment]
@choudrio I don't believe there is much activity on the angular-meteor project at the moment, we are also trying to get more information about the status of the stack and the options for further support in the future.

Don't hesitate to keep me posted if you have any news on your side.

**Comment by Urigo** [comment]
today we've released new versions for `angular-compilers` and `meteor-client-bundler` with a few fixes and to support the latest Meteor 1.6.1, thanks to the great work of @ardatan and @afaur !
Let us now if everything works as expected.

**Comment by Urigo** [comment]
as a side note, my personal email is public and I'm always happy to talk, hear and help users of angular-meteor with their questions, whatever questions they may be!
